src/Table/Table.test.tsx

```diff
@@ -144,7 +144,7 @@ describe("Component: Table", () => {
     it("Should render and do pagination where necessary ", async () => {
         const pageSize: number = 30;
         const paginationValue: number = 1;
-        const setPage: jest.Mock = jest.fn((n: number) => console.log("setPage called"));
+        const setPage: jest.Mock = jest.fn((n: number) => true);
```

Comment:
i think this can be simplified with `jest.fn()`

Reply:
Done